edge/pkg/metamanager/client/configmap.go

```diff
@@ -31,10 +31,10 @@ type configMaps struct {
 	send      SendInterface
 }
 
-func newConfigMaps(namespace string, s SendInterface) *configMaps {
+func newConfigMaps(ns string, s SendInterface) *configMaps {
 	return &configMaps{
 		send:      s,
-		namespace: namespace,
```

Comment:
Namespace is needed.

Reply:
Here i do not delete them, just rename them.

Reply:
The namespace no need rename to name.